stable/metrics/templates/deployment.yaml

```diff
@@ -54,10 +56,14 @@ spec:
             value: "{{ .Values.conf.devHoseURL }}"
           - name: DEVHOSE_TENANT
             value: "{{ .Values.conf.devHoseTenant }}"
-          - name: LEDGER_SOURCE
-            value: "{{ .Values.conf.ledgerSource }}"
+          - name: DEVHOSE_METRICS_SOURCE
+            value: "{{ .Values.conf.devHoseMetricsSource }}"
+          - name: DEVHOSE_FINDINGS_SOURCE
+            value: "{{ .Values.conf.devHoseFindingsSource }}"
           - name: REDIS_ADDR
             value: "{{ .Values.conf.redisAddr | default (include "metricsRedisAddr" .)}}"
+          - name: VULCAN_API_BASE_URL
```

Comment:
I would change this and default to the calculated VULCAN_API internal endpoint, this way you don't have to set any value for that link.

You can see an example in the continuous component.

Reply:
It should be done now.